pynest/nest/tests/test_connect_arrays.py

```diff
@@ -20,29 +20,52 @@
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
 import unittest
-import nest
 import numpy as np
 
+import nest
+from nest.lib.hl_api_exceptions import NESTErrors
+
 nest.set_verbosity('M_WARNING')
 
 
+non_unique = np.array([1, 1, 3, 5, 4, 5, 9, 7, 2, 8], dtype=np.uint64)
```

Comment:
Move `non_unique` from global scope into `TestConnectArrays`, and use it as `self.non_unique`.